web/index.js

```diff
@@ -80,17 +79,20 @@ map.on('load', function() {
         }).setLngLat(e.lngLat).setHTML(html).addTo(map);
     };
 
-    map.on('click', 'stop-label-intercity', on_stop);
-    map.on('click', 'stop-label-urban', on_stop);
+    map.on('click', 'stops-intercity-icon', on_stop);
+    map.on('click', 'stops-urban-icon', on_stop);
 });
 
 
 function filter_on_one_route(route) {
     //route_id = -7773405
```

Comment:
this comment is deprecated as the parameter of the function is not a route_id anymore, we should remove it.

Reply:
ok